src/Applications/LDAS_App/sample_config_files/LADAS/HISTORY.rc.atmens

```diff
@@ -0,0 +1,1114 @@
+
+VERSION: 1
+EXPID:  ldas1711_C90
+EXPDSC: GEOSldas_output
+EXPSRC: GEOSldas
+
+COLLECTIONS:
```

Comment:
@saraqzhang: 
Has this HISTORY.rc file been generated with the help of `generate_catchincr_hist.py`?  If so, please add a comment into the sample HIST rc file that mentions the script. 
Also, the "catch_progn_incr" output is in tile space and the associated HIST.rc entries are therefore independent of the resolution of the LADAS.  Only the "lndfcstana" output is explicitly linked to the LADAS resolution.  It would probably be better to
(i) define "lndfcstana" **before** the long list of "catch_progn_incr" definitions in the sample HIST rc file, 
(ii) add a comment that the "lndfcstana" resolution should be adjusted to be consistent with that of the LADAS, and
(iii) add a comment that the "catch_progn_incr" output is in tile space and its definition is generic for any LADAS resolution.

Reply:
Thanks, @saraqzhang!  I'll take another look at the updates tomorrow and might tweak the wording a bit.
Re. (1): Correct, the sample files in this directory should not be installed.  No need for CMake. 
Re. (2): It would have been a bit easier to trace if the old expinp.txt*C90 and *C180 files had been renamed (e.g., `git mv exeinp.txt.atmensC90 exeinp.txt.Hy4dEnVar.atmens`) and then edited.  Since these are new files it's not important here, but in future commits, if we're changing a file name, I would recommend "git mv oldfilename newfilename", rather than deleting the old file outright.